posthog/clickhouse/system_status.py

```diff
@@ -128,6 +120,26 @@ def is_alive() -> bool:
         return False
 
 
+def dead_letter_queue_ratio() -> Tuple[bool, int]:
+    dead_letter_queue_events_last_day = get_dead_letter_queue_events_last_24h()
+
+    total_events_ingested_last_day = sync_execute(
+        "SELECT count(*) as b from events WHERE _timestamp >= (NOW() - INTERVAL 1 DAY)"
+    )[0][0]
+
+    dead_letter_queue_ingestion_ratio = dead_letter_queue_events_last_day / max(
+        dead_letter_queue_events_last_day + total_events_ingested_last_day, 1
+    )
+
+    # if the dead letter queue has as many events today as ingestion, issue an alert
```

Comment:
nit: let's update this outdated comment? The comment implies we alert at 100% DLQ, while we alert on 20% (which is more reasonable)